app/controllers/catalog_controller.rb

```diff
@@ -160,12 +160,13 @@ class CatalogController < ApplicationController
       pc.add_show_field 'subject_label_tsim', label: 'Subject'
       pc.add_show_field 'link_ssim', label: 'Link'
       pc.add_show_field 'publisher_label_tsim', label: 'Publisher'
-      pc.add_show_field 'Grants', label: 'Grant', accessor: :linked_grants
+      pc.add_show_field 'grants', label: 'Grant', accessor: :linked_grants
```

Comment:
Should the label here be `Grants`, or are you thinking publications will be associated with only one grant?

Reply:
I suspect most will only have one grant...but we'll find out.

Reply:
Easy change to make later when we know more. :+1: